django/contrib/admin/checks.py

```diff
@@ -19,9 +19,11 @@
 
 
 def check_admin_app(**kwargs):
-    from django.contrib.admin.sites import system_check_errors
-
-    return system_check_errors
+    from django.contrib.admin.sites import system_checks
```

Comment:
I think it would make more sense to drop the entire method and register all checks separately in the admin's AppConfig